neo/CMakeLists.txt

```diff
@@ -336,6 +336,9 @@ elseif(MSVC)
 	
 endif()
 
+SET(CMAKE_MAP_IMPORTED_CONFIG_RETAIL RELEASE)
+SET(CMAKE_MAP_IMPORTED_CONFIG_RETWITHDEBUGSYMBOLS RELEASE)
```

Comment:
Upper-case RETAIL is what I noticed in the docs, too.
I would move these lines up to the location where the project defines the meaning of "retail" et al.